practice-one/pages/index.html

```diff
@@ -64,7 +64,7 @@ <h2 class="section-title">What We Do</h2>
           </div>
           <div class="card-body">
```

Comment:
This is a section for services the web app will have. So I think, you should use the prefix **service** in the way to naming

Reply:
So if I named it like that I just can use it for this section?